### ISSUE TITLE
[improvement] Confluence companion as systemd service.

### ISSUE THREAD

**Issue by WatchTh1** [schorschii/companion-linux#2]
I'd suggest to make a systemd unit for companion.py and move configuration into $HOME/.config/companion/*.conf

So multiuser environment would be supported.

**Comment by schorschii** [comment]
I think the easiest solution is the current one with the `companion.desktop` file in `/etc/xdg/autostart`. Systemd has support for user sessions but that means that the service file must be copied into every users home directory (`$HOME/.local/share/systemd/user/companion.service`). Using systemd has no advantages in this case from my point of view.

Multiuser support is difficult in general because of the Confluence API design. The client has to listen on Port 31459 for incoming connections from the web browser. If a second user logs in, the script can not open the port because it is already opened by the first user. This is especially a problem in terminal server environments and of course also persists in the official clients for Windows and macOS.

TLDR; Only one user can use the companion app at the same time.

**Comment by WatchTh1** [comment]
Well, there are several usecases about multiuser envs. 
1st one - is terminal server. Which is ofc a problem. I think embedding router proxy into companion should solve this issue. Every session has enough data to distinguish one from another, but this is sure to be compliacated, but usefull usecase.
2nd one - is an environment with single user to be logged in simultaneously but several to work over time.

Second usecase could be solved through managing per user configuration. In future it could help to solve 1st one as well (multiple instances of companion run simultaneously in termnial server behind router proxy). 

Per user configuration gives one more advantage, you don't need to be root to add confluence site to trusted list. And thus when you will make a GUI you can easily manage trusted sites with it run from regular linux user environment.
